src/isar/mission_planner/local_planner.py

```diff
@@ -29,7 +29,7 @@ def get_mission(self, mission_id) -> Mission:
             mission: Mission = missions[mission_id]["mission"]
             mission.set_unique_id_and_metadata()
             return mission
-        except Exception as e:
+        except KeyError as e:
```

Comment:
Do we still need to catch general exception and throw error accordingly?

Reply:
Added catching the general exception in addition